packages/sdk/governance/src/proposals.ts

```diff
@@ -287,37 +289,44 @@ export class ProposalBuilder {
   getRegistryAddition = (contract: CeloContract): string | undefined =>
     this.registryAdditions[stripProxy(contract)]
 
-  isRegistered = (contract: CeloContract) =>
+  isRegistryContract = (contract: CeloContract) =>
```

Comment:
I wonder if we can keep this name the same

Reply:
aliased